examples/formbot/train.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from rasa_core import utils
+from rasa_core.agent import Agent
+from rasa_core.policies.keras_policy import KerasPolicy
+from rasa_core.policies.memoization import AugmentedMemoizationPolicy
+from rasa_core.policies.form_policy import FormPolicy
+from rasa_core.policies.fallback import FallbackPolicy
+
+if __name__ == '__main__':
+    utils.configure_colored_logging(loglevel="INFO")
+
+    training_data_file = 'data/stories.md'
+    model_path = 'models/dialogue'
+
+    agent = Agent("domain.yml",
+                  policies=[AugmentedMemoizationPolicy(),
+                            FormPolicy(),
```

Comment:
we can add the formpolicy by default though, right? (so you do not need to customize this if you want to use forms)

Reply:
what do you mean? add to `train.py`, or always add it?

Reply:
yeah i'd say so, no reason not to always include it in train.py